src/main/docker/Dockerfile.jre

```diff
@@ -24,8 +24,17 @@ ADD https://raw.githubusercontent.com/CAFapi/caf-common/v1.14.0/container-cert-s
     /startup/startup.d/
 RUN chmod +x /startup/startup.d/install-ca-cert-java.sh
    
-# Install OpenJDK 8 Runtime Environment
-RUN zypper -n install java-1_8_0-openjdk
+# Install OpenJDK 8 Runtime Environment and Patch
+RUN zypper -n install java-1_8_0-openjdk patch
+
+# Set Java Home
+ENV JAVA_HOME=/usr/lib64/jvm/java-1.8.0-openjdk-1.8.0
+
+# Disable weaker security algorythms
+COPY 0001-prevent-Logjam-disableAlgorythms.patch $JAVA_HOME/jre/lib/security/
+RUN cd $JAVA_HOME/jre/lib/security && \
+    patch -i 0001-prevent-Logjam-disableAlgorythms.patch java.security && \
```

Comment:
This patch is for specific openjdk release (8u232), it may not work with further releases.
So, this image build would be broken unexpectedly by new openjdk releases.
Using of exact openjdk release in dockerfile makes build reproducible.

Reply:
So long as it breaks, I don't think we need to worry about it.  The builds will be run regularly and a sudden unexpected build break would automatically cause a Jira to be raised and would be fixed immediately.

However I would be concerned if there is a possibility that it stops doing what it is intended to do without causing a build break.  Is that a possibility?  If it is then we'll have to introduce some tests to ensure that the algorithms have been disabled as intended.

Reply:
> However I would be concerned if there is a possibility that it stops doing what it is intended to do without causing a build break. Is that a possibility?

No